src/cf-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "3.4.0",
+  "version": "3.6.0",
```

Comment:
Core is getting bumped by two minor version increments, because it's listed twice in the original chance log (one change, one removal). Can we change the script so that it only increments by one in such cases?